src/Mvc/Mvc.sln

```diff
@@ -274,6 +274,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.DataPr
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GenericHostWebSite", "test\WebSites\GenericHostWebSite\GenericHostWebSite.csproj", "{D9BE3E50-5CE8-4D8D-BA19-AA219D009752}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components", "..\Components\Components\src\Microsoft.AspNetCore.Components.csproj", "{9C933F42-C167-4C51-96C8-CE9BF3B6F852}"
```

Comment:
?

Reply:
It fixes a NuGet error caused by a referenced project not being in the solution. I get it because I'm still using VS2019 preview 1 instead of dog food build. I just happened to have fixed it before working on this PR

Reply:
... preview1 😢